tools/tdbg/commands.go

```diff
@@ -121,13 +128,115 @@ func AdminShowWorkflow(c *cli.Context) error {
 
 	if outputFileName != "" {
 		encoder := codec.NewJSONPBEncoder()
-		data, err := encoder.EncodeHistoryEvents(allEvents.Events)
+		data, err := encoder.EncodeHistories(historyBatches)
 		if err != nil {
 			return fmt.Errorf("unable to serialize History data: %s", err)
 		}
 		if err := os.WriteFile(outputFileName, data, 0666); err != nil {
-			return fmt.Errorf("unable to export History data file: %s", err)
+			return fmt.Errorf("unable to write History data file: %s", err)
+		}
+	}
+	return nil
+}
+
+// AdminImportWorkflow shows history
+func AdminImportWorkflow(c *cli.Context) error {
+	nsName, err := getRequiredOption(c, FlagNamespace)
+	if err != nil {
+		return err
+	}
+	wid, err := getRequiredOption(c, FlagWorkflowID)
+	if err != nil {
+		return err
+	}
+	rid, err := getRequiredOption(c, FlagRunID)
```

Comment:
Is this required? Will the history contains the run id?

Reply:
this is required, the history will only contain next / prev run ID.

i also believe the next / prev run ID should be excluded from history